Assignment2/BasilisKing-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,40 @@
+{
+  "@context":{
+
+    "base":  "http://www.example.net/ontologies/",
+	"rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "s": "http://www.example.net/ontologies/sensor",
+    "m": "http://www.example.net/ontologies/measurements",
+    "c" : "http://www.example.net/ontologies/computer",
+    "u" : "http://www.example.net/ontologies/users",
+    "sensor": "s:sensor",
+    "measurement": "m:measurement",
+    "value": "measurement:value",
+    "atTime": "measurement:Time",
+    "computer": "c:computer",
+    "user": "u:user",
+    "name": "rdfs:label"
+  },
```

Comment:
You have to define the properties "includes", "hasOwner", "hasMeasurement"
Check the structure of the file because you will not need defining "sensor":, "measurement": , "computer": and "user": All they are resources, thus you don't need them as attributes.